### PR TITLE
chore: use updater in Livechat Contacts' `updateContactChannel`

### PR DESCRIPTION
**Pull request by matheusbsilva137**
## Proposed changes (including videos or screenshots)

 - Split `updateContactChannel` in multiple methods and use updater to avoid multiple DB trips.

## Further comments

[SCI-174]

[SCI-174]: https://rocketchat.atlassian.net/browse/SCI-174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by matheusbsilva137 — COMMENTED** [review]
Keeping this PR as a draft for now so that we don't accidentally merge this and delay the main PR. This is ready for review, though

File: apps/meteor/app/livechat/server/lib/Helper.ts [hunk 1]
```diff
@@ -95,9 +95,14 @@ export const createLivechatRoom = async (
 	});
 
 	const source = extraRoomInfo.source || roomInfo.source;
+	const updater = LivechatContacts.getUpdater();
```
Comment (MarcosSpessatto) [inline apps/meteor/app/livechat/server/lib/Helper.ts:98]: Is there any benefit of using the updater here? 🤔

File: apps/meteor/ee/app/livechat-enterprise/server/api/lib/contacts.ts [hunk 2]
```diff
@@ -6,7 +6,10 @@ import { Livechat } from '../../../../../../app/livechat/server/lib/LivechatType
 import { i18n } from '../../../../../../server/lib/i18n';
 
 export async function changeContactBlockStatus({ block, visitor }: { visitor: ILivechatContactVisitorAssociation; block: boolean }) {
-	const result = await LivechatContacts.updateContactChannel(visitor, { blocked: block });
+	const updater = LivechatContacts.getUpdater();
```
Comment (MarcosSpessatto) [inline apps/meteor/ee/app/livechat-enterprise/server/api/lib/contacts.ts:9]: Is there any benefit of using the updater here? 🤔

File: apps/meteor/ee/server/patches/verifyContactChannel.ts [hunk 3]
```diff
@@ -29,20 +29,10 @@ async function _verifyContactChannel(
...
-				verifiedAt: new Date(),
-				field,
-				value: value.toLowerCase(),
-			},
-			{},
-			{ session },
-		);
+		const updater = LivechatContacts.getUpdater();
+		LivechatContacts.setVerifiedUpdateQuery(true, updater);
+		LivechatContacts.setFieldAndValueUpdateQuery(field, value, updater);
```
Comment (MarcosSpessatto) [inline apps/meteor/ee/server/patches/verifyContactChannel.ts:34]: Aren't we forgetting the lower case here?

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]

File: apps/meteor/ee/server/patches/verifyContactChannel.ts (hunk 3 above)
Comment (matheusbsilva137) [reply apps/meteor/ee/server/patches/verifyContactChannel.ts:34]: hmmm not sure I got you... which lower case?

**Review by KevLehman — APPROVED** [review]

**Pull request merged**